autoscale/src/main/java/com/sequenceiq/periscope/service/EntitlementValidationService.java

```diff
@@ -0,0 +1,44 @@
+package com.sequenceiq.periscope.service;
+
+import java.util.Map;
+import java.util.Optional;
+
+import javax.inject.Inject;
+
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.cache.annotation.Cacheable;
+import org.springframework.stereotype.Service;
+
+import com.sequenceiq.cloudbreak.auth.altus.EntitlementService;
+
+@Service
+public class EntitlementValidationService {
+
+    private static final String ALLOWED = "ALLOWED_PLATFORM";
+
+    private static final Map<String, String> PLATFORM_ENTITLEMENTS = Map.of(
+            "AWS", "CDP_AWS_AUTOSCALING",
```

Comment:
Names changes to DATAHUB_ instead of CDP_

Reply:
ack